index.css

```diff
@@ -0,0 +1,318 @@
+body{
+    margin: 0;
+    padding: 0;
+    width: 100vw;
+    overflow-x: hidden;
+    /* position: relative; */
+    
+}
+#header{
+    margin: 0;
+    /* position: sticky; */
+    padding: 10px 20px;
+    /* background-color: aqua; */
+    display: flex;
+    width: 100vw;
+    /* overflow-x: hidden; */
+    /* align-items: center;
+    justify-content: center; */
+}
+.icon{
+    height: 25px;
+    background-color:rgba(236, 233, 233, 0.787) ;
+    border-radius: 50%;
+    padding: 5px;
+    margin: auto 10px;
+}
+.icon:hover{
+    cursor: pointer;
+}
+#menu-logo{
+    height: 40px;
+    position: relative;
+}
+#menu-logo img{
+    border: none;
+    border-radius: 0;
+    background-color: white;
+}
+nav{
+    display: none;
+    flex-direction: column;
+    position: absolute;
+    top: 60px;
+    background-color: white;
+    line-height: 65px;
+    padding: 5px;
+    /* z-index: 999; */
+    width: 200px;
+    height: 150vh;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    align-items: flex-start;
+}
+a{
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+}
+nav *{
+    text-decoration: none;
+    font-size: 18px;
+    color: rgba(0, 0, 0, 0.751);
+    float: left;     
+}
+nav a:hover{
+    background-color: lightgray;
+    border-radius: 5px;
+    width: 200px;
+    height: 40px;
+
+}
+.nav-icon{
+    width: 25px;
+    background-color:lightgray;
+}
+hr{
+    border-bottom: 0.01px solid rgba(211, 211, 211, 0.019);
+    width: 90%;
+    margin: 10px auto;
+}
+#youtube-logo{
```

Comment:
consider removing all whitespace within your rulesets and leaving a new line after each ruleset